monkey/monkey_island/cc/services/agent_signals_service.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from monkey_island.cc.models import Agent, AgentSignals
+from monkey_island.cc.repository import ISimulationRepository
+
+
+class AgentSignalsService:
+    def __init__(self, simulation_repository: ISimulationRepository):
+        self._simulation_repository = simulation_repository
+
+    def get_signals(self, agent: Agent) -> AgentSignals:
+        """
+        Stubbed method
+        """
+        return AgentSignals(timestamp=datetime.now())
+
+    def on_terminate_agents_signal(self, timestamp: datetime):
+        """
+        Stubbed method
+        """
+        pass
```

Comment:
We can add the docstrings right now. We know what these methods are going to do.

Reply:
+1